frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -189,10 +237,31 @@ export const IndividualResponsePage = (): JSX.Element => {
           </Text>
           <Box display="inline-flex">
             <Text as="span" textStyle="subhead-1">
-              Time:&nbsp;
+              Timestamp:
+            </Text>
+            <Skeleton isLoaded={!isLoading && !isError}>
+              &nbsp;{data?.submissionTime ?? 'Loading...'}
+            </Skeleton>
+          </Box>
+          <Box
+            alignItems="center"
+            display="inline-flex"
+            hidden={attachmentDownloadUrls.size === 0}
+          >
+            <Text as="span" textStyle="subhead-1">
+              Attachments:
             </Text>
             <Skeleton isLoaded={!isLoading && !isError}>
-              {data?.submissionTime ?? 'Loading...'}
+              &nbsp;
+              <Button
+                variant="link"
+                aria-label="Download attachments as ZIP"
```

Comment:
is there a need for an aria-label seeing how there is already button text inside the button?